src/main/kotlin/com/devfloor/untitled/article/application/ArticleFacade.kt

```diff
@@ -12,8 +15,30 @@ class ArticleFacade(
     private val articleService: ArticleService,
     private val articleHashtagService: ArticleHashtagService,
     private val articleFavoriteService: ArticleFavoriteService,
-    private val articleOptionService: ArticleOptionService
+    private val articleOptionService: ArticleOptionService,
+    private val optionService: OptionService,
+    private val hashtagService: HashtagService
 ) {
+    @Transactional
+    fun create(articleRequest: ArticleRequest, user: User): Long {
+        val article = articleService.create(articleRequest.toArticleWithUser(articleRequest, user))
+        if (!articleRequest.options.isNullOrEmpty()) {
+            optionService.showAllByOptionType(articleRequest.options)
+                .let { articleOptionService.createAllByOptions(article, it) }
+        }
+        if (!articleRequest.hashtags.isNotEmpty()) {
```

Comment:
이 부분 혹시 `!`과 `Not`을 없애고 아래와같이 변경하면 안되는 점이 있나요!??
```suggestion
        if (articleRequest.hashtags.isEmpty()) {
```

Reply:
없습니다! 수정하겠습니다!